addons/website_event_track/controllers/main.py

```diff
@@ -159,4 +160,11 @@ def event_track_proposal_post(self, event, **post):
             partner = request.env['res.partner'].sudo().search([('email', '=', post['email_from'])])
             if partner:
                 track.sudo().message_subscribe(partner_ids=partner.ids)
+        return request.redirect('/event/track_proposal/%s/%s/success' % (track.id, event.id))
+
+    @http.route(['/event/track_proposal/<model("event.track"):track>/<model("event.event"):event>/success'], type='http', auth="public", website=True)
```

Comment:
Same here, change route to comply with best practices the same way as my previous comment.

Reply:
Changes Done.